ocean_data_gateway/__init__.py

```diff
@@ -1,34 +1,37 @@
+import logging
+from pathlib import Path
+
 try:
     from ._version import __version__
 except ImportError:
     __version__ = "unknown"
 
 
-## make necessary directories ##
-
-# make base dir in user home directory
-import pathlib
-path_base = pathlib.Path.home().joinpath('.ocean_data_gateway')
-path_base.mkdir(parents=True, exist_ok=True)
-
-# make subdirectories too
-path_catalogs = path_base.joinpath('catalogs')
-path_catalogs.mkdir(parents=True, exist_ok=True)
-path_logs = path_base.joinpath('logs')
-path_logs.mkdir(parents=True, exist_ok=True)
-path_variables = path_base.joinpath('variables')
-path_variables.mkdir(parents=True, exist_ok=True)
-
-
-# import search.search
-from .readers import erddap, axds, local
-from .gateway import Gateway
-# import ocean_data_gateway.readers.erddap
-# import readers.axds
-# import readers.local
-#
-# from .gateway import (gateway)
-# import Data
-# from .ErddapReader import (ErddapReader, region)
-# from .axdsReader import (axdsReader)
-# from .localReader import (localReader)
+base_path = Path.home() / ".ocean_data_gateway"
+base_path.mkdir(exist_ok=True)
+
+logs_path = base_path / "logs"
+logs_path.mkdir(exist_ok=True)
+
+catalogs_path = base_path / "catalogs"
```

Comment:
It's hard to look through all of these formatting changes to see — did you update all of these names in the respective files to be consistent?

Reply:
Yeah, sorry for conflating the renaming there with what was supposed to be focused on installation and the ability to run tests.  Renaming is consistent throughout the package.

Reply:
👍